tracer/src/Datadog.Trace/DiagnosticListeners/RequestDelegate.cs

```diff
@@ -0,0 +1,23 @@
+// <copyright file="RequestDelegate.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.DuckTyping;
+
+#nullable enable
+
+namespace Datadog.Trace.DiagnosticListeners;
+
+/// <summary>
+/// RequestDelegate for duck typing
+/// https://github1s.com/dotnet/aspnetcore/blob/v3.0.3/src/Http/Http.Abstractions/src/RequestDelegate.cs
```

Comment:
```suggestion
/// https://github.com/dotnet/aspnetcore/blob/v3.0.3/src/Http/Http.Abstractions/src/RequestDelegate.cs
```

Reply:
Fixed in adcb1b8cd07a697feb8e891a1a4f74b39b6d5d5a